benchmarks/Elastic.Transport.Benchmarks/Elastic.Transport.Benchmarks.csproj

```diff
@@ -3,11 +3,12 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <IsPackable>false</IsPackable>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
```

Comment:
There is a bug in 0.12.1 which means that we cannot use latest here from `Directory.Build.props` so temporarily fixing this to 9.0.

Once Benchmark.NET includes a fix, we can remove this after updating to that version.

Reply:
Is there already an open a ticket for this on Benchmark.NET ?

Reply:
Yep, it's fixed and will be in 0.12.2 - https://github.com/dotnet/BenchmarkDotNet/pull/1420